topics/eleventy/index.md

```diff
@@ -0,0 +1,13 @@
+---
+aliases: 11ty
+created_by: Zach Leatherman
+display_name: Eleventy
+github_url: https://github.com/11ty
+logo: eleventy.png
+related: jekyll
```

Comment:
For related, could we change this to words that describe what the project does as opposed to similar products?

Reply:
How about `static-site-generator`?